node-tests/helpers/expect-coffee.js

```diff
@@ -0,0 +1,10 @@
+var coffeescript = require('coffee-script');
+var expect = require('ember-cli-blueprint-test-helpers/chai').expect;
+
+module.exports = function(file) {
+  var compileFunc = function() {
+    coffeescript.compile(file.content);
+  }
+
+  expect(compileFunc).to.not.throw(Error);
```

Comment:
Running the literal `expectCoffee('asjdnasdjk');` might throw an error for the wrong reasons though - on line 6 it would call `'asjdnasdjk'.content` and get `undefined`, which is then passed into `coffeescript.compile()`. Would you mind testing by passing in a random string to `coffeescript.compile()` directly instead?